src/fed_dbm.cpp

```diff
@@ -97,10 +98,11 @@ namespace dbm
         }
 
         cindex_t dim = pdim();
-        uint32_t mingraph[bits2intsize(dim * dim)];
+        std::vector<uint32_t> mingraph = std::vector<uint32_t>(bits2intsize(dim * dim));
```

Comment:
consider: `auto mingraph = std::vector<uint32_t>(bits2intsize(dim * dim));`

Reply:
when I look at the usage of mingraph, it seems that we should use `std::vector<bool>` and get rid of `base_setOneBit`, I am just not sure if `std::vector<bool>::data()` uses the same encoding as `dbm_cleanBitMatrix`